recipe/meta.yaml

```diff
@@ -1,38 +1,55 @@
 {% set name = "bson" %}
-{% set version = "0.5.9" %}
-{% set sha256 = "84a06dd1fa350d5814d9845abad982e8ac4c9768c4f41afeaa5fdb6d58570158" %}
+{% set version = "0.5.10" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  git_url: https://github.com/py-bson/bson.git
```

Comment:
This makes me feel nervous because they didn't create a real release. I think it would be safer to use https://github.com/py-bson/bson/archive/refs/tags/0.5.10.tar.gz (the archive generated by GH on the tag) since we would at least be able to checksum the file. I'll also make the build faster (because it's usually faster to download an artifact than cloning and checkout out a repo).